src/components/modals/VideoCallModal.tsx

```
@@ -11,75 +13,157 @@ export default function VideoCallModal({ isOpen, onClose, roomId }: VideoCallMod
+
+        pc = new RTCPeerConnection({
+          iceServers: [
+            { urls: 'stun:stun.l.google.com:19302' },
+            { urls: 'stun:stun1.l.google.com:19302' }
+          ]
+        });
         peerConnectionRef.current = pc;
-        stream.getTracks().forEach(track => pc.addTrack(track, stream));
+
+        // Add local tracks to peer connection
+        stream.getTracks().forEach(track => pc?.addTrack(track, stream));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)